frontend/src/components/admin/RoundBuildingCard.vue

```diff
@@ -88,7 +88,7 @@ export default {
       // TODO opslaan in database
     },
     goToBuildingPage: function () {
-      router.push({ path: 'building/' + this.data.id })
+      router.push({ path: '/building/' + this.data.id })
```

Comment:
Kan ook ``router.push({name: 'admin_info_building', params: {id: this.data.id})``

Reply:
We zullen nog kijken voor milestone 3 dat het allemaal ongeveer op dezelfde manier gebeurt, maar het werkt voor nu